apps/session-manager/src/repositories/redis.ts

```diff
@@ -0,0 +1,180 @@
+import * as redis from "redis";
+import * as appInsights from "applicationinsights";
+import { pipe } from "fp-ts/lib/function";
+import * as RA from "fp-ts/lib/ReadonlyArray";
+import * as O from "fp-ts/lib/Option";
+import { log } from "../utils/logger";
+
+export const sessionKeyPrefix = "SESSION-";
+export const walletKeyPrefix = "WALLET-";
+export const myPortalTokenPrefix = "MYPORTAL-";
+export const bpdTokenPrefix = "BPD-";
+export const zendeskTokenPrefix = "ZENDESK-";
+export const fimsTokenPrefix = "FIMS-";
+export const userSessionsSetKeyPrefix = "USERSESSIONS-";
+export const sessionInfoKeyPrefix = "SESSIONINFO-";
+export const noticeEmailPrefix = "NOTICEEMAIL-";
+export const blockedUserSetKey = "BLOCKEDUSERS";
+export const lollipopDataPrefix = "KEYS-";
+export const keyPrefixes = [
+  sessionKeyPrefix,
+  walletKeyPrefix,
+  myPortalTokenPrefix,
+  fimsTokenPrefix,
+  bpdTokenPrefix,
+  zendeskTokenPrefix,
+  userSessionsSetKeyPrefix,
+  sessionInfoKeyPrefix,
+  noticeEmailPrefix,
+  blockedUserSetKey,
+  lollipopDataPrefix,
+];
```

Comment:
I would make this array readonly
```suggestion
export const keyPrefixes = [
  sessionKeyPrefix,
  walletKeyPrefix,
  myPortalTokenPrefix,
  fimsTokenPrefix,
  bpdTokenPrefix,
  zendeskTokenPrefix,
  userSessionsSetKeyPrefix,
  sessionInfoKeyPrefix,
  noticeEmailPrefix,
  blockedUserSetKey,
  lollipopDataPrefix,
] as const;
```